### ISSUE TITLE
Accessing template refs for recursive sortable

### ISSUE THREAD

**Issue by nlhkabu** [MaxLeiter/sortablejs-vue3#41]
I am using this library to build a nested list, using a recursive component.

You can see a demo of this here: https://stackblitz.com/edit/recursive-template-refs-b2umvk?file=src/components/EntryListItem.vue

Each `entryListItem` has a `focusText` method that I want to be able to call from anywhere by providing the entry ID

(one use case is that the user presses the up or down arrows on their keyboard and the focus is moved to the next item, regardless of indentation)

To do this I think I need to build a list of template refs, but I am struggling to do this because of the recursion. Any ideas on how I should move forward on this? Is this something that could/should be handled at this library's level?

Thanks in advance!

**Comment by MaxLeiter** [comment]
I played with your stackblitz (thank you for the repro 🙏 ) to find a user-space solution but couldn't quite get it to work.

I agree this is a place the library could help by maintaining an internal list of refs, which could be combined to cleanup the DOM after insertion/deletion (i.e. #40). 

I'm not sure when I'll have time to work on it, but does that sound correct to you? What do you think a useful API would look like for your use-case?

**Comment by nlhkabu** [comment]
Hi @MaxLeiter thanks for your quick response to this and sorry for my delayed reply! :)  In the end we have found a solution where we store the refs in Pinia.  Thinking about this more deeply: I'm not sure how useful building in ref support to this lib would be for our use case, as we needed to access the component _inside_ the list item.
I'm going to close this as a result.
Thanks again for your time.
